README.md

```diff
@@ -30,12 +38,52 @@ services:
       - cron.another_task.schedule="*/2 * * * *"
       - cron.another_task.command="/usr/local/bin/another_app_script.sh"
 ```
+This example shows how to schedule multiple cron jobs using cron syntax. The Docker container will run `/usr/local/bin/app_script.sh` every minute and `/usr/local/bin/another_app_script.sh` every two minutes, with logs stored in `/var/log/cron/`.
 
 ### Environment Variables
-
 The `example_compose.yml` file uses several environment variables. Make sure to define these in a `.env` file in the root directory of your project:
+- **`COMPOSE_PROJECT_NAME`**: Specifies the name of the Docker Compose project, allowing the cron to target jobs in that specific project.
```

Comment:
I think we should note that if `COMPOSE_PROJECT_NAME` is not defined, the cron container will process the jobs from all the docker-compose stacks on the computer, the cron jobs will be executed simultaneously as many times, as many cron containers with no `COMPOSE_PROJECT_NAME` defined are running